Rubberduck.Core/Navigation/CodeExplorer/CodeExplorerMemberViewModel.cs

```diff
@@ -76,10 +76,14 @@ protected sealed override void AddNewChildren(List<Declaration> updated)
         {
             if (updated != null)
             {
-                AddChildren(updated
-                    .Where(item =>
-                        SubMemberTypes.Contains(item.DeclarationType) && item.ParentDeclaration.Equals(Declaration))
-                    .Select(item => new CodeExplorerSubMemberViewModel(this, item)));
+                var updates = updated.Where(item => SubMemberTypes.Contains(item.DeclarationType) && item.ParentDeclaration.Equals(Declaration)).ToList();
```

Comment:
I think the line is too wide and should be broke up.

Reply:
Done. This is what happens after R# crashes. _Curse you Visual Studio!!!_